packages/integration-sdk-runtime/src/storage/memory.ts

```diff
@@ -203,6 +217,7 @@ export class InMemoryGraphObjectStore implements GraphObjectStore {
 
       entityTypeKeysMap.delete(entity._key);
     }
+    this.entitySizeInBytes[stepId] = 0;
```

Comment:
Not a huge deal, but it might be better to delete `this.entitySizeInBytes[stepId]` rather than set to `0` - AWS, for example, has 399 steps, so by the end of the integration, the `getTotalEntitySizeInBytes()` call will be summing 399 counts, most of which will be `0`. I think this operation occurs on every `jobState.addEntity()` call.

Maybe you can look into the performance of `getTotalEntitySizeInBytes()` when its an object with 1 property compared to an object of 400 properties.